config/certmanager/certificate.yaml

```diff
@@ -1,23 +1,23 @@
 # The following manifests contain a self-signed issuer CR and a certificate CR.
 # More document can be found at https://docs.cert-manager.io
-apiVersion: certmanager.k8s.io/v1alpha2
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
Why the version downgrade?

Reply:
certmanager is actually in v1alpha1 https://github.com/jetstack/cert-manager/blob/master/deploy/manifests/00-crds.yaml